Documentation/using-corert/debugging-corert.md

```diff
@@ -0,0 +1,13 @@
+# Debugging CoreRT programs
```

Comment:
Nit: Should this doc be linked from somewhere to make it discoverable?

Reply:
I thought I would write a README.md for the using-corert directory and link it from there, but I don't want to keep this hanging on that.